content/blogposts/2022/code-intelligence-platform.md

```diff
@@ -0,0 +1,35 @@
+---
+title: "From code search to a code intelligence platform"
+publishDate: 2022-09-27T10:00-07:00
+description: No matter how good your dev tools are, coding is complex. We're building a code intelligence platform to help you unblock yourself and code better.
+tags: [blog]
+slug: "code-search-to-code-intelligence"
+published: false
+heroImage: https://storage.googleapis.com/sourcegraph-assets/blog/4.0/to-code-intelligence.png
+socialImage: https://storage.googleapis.com/sourcegraph-assets/blog/4.0/to-code-intelligence.png
+---
+
+No matter how good your dev tools are, coding is complex. This is true even if you've configured your editor just right, sped up your test suite, set up continuous deployment, and added observability. Higher-level languages, libraries, and frameworks can help, but as you take on bigger and more ambitious projects, you'll likely spend your complexity budget elsewhere. The overall complexity of code has increased over time, and there's no sign of that trend changing.
```

Comment:
@sqs: Do you have a stat about the amount of code being written year-over-year increasing? That would add credibility to this statement.

I believe you had a stat you used at Marge, but I haven't been able to find it.

Reply:
Nope. I just have what was in the Big Code survey from a couple years ago. Anything else is a guess.